src/hooks/useSearchSubmit.ts

```diff
@@ -1,18 +1,15 @@
 import { useCallback } from 'react'
-import { useHistory, useRouteMatch } from 'react-router-dom'
+import { useHistory } from 'react-router-dom'
 
 export function useSearchSubmit(): (query: string) => void {
   const history = useHistory()
-  const { path } = useRouteMatch()
 
   return useCallback(
     (query: string) => {
-      const pathPrefix = path.endsWith('/') ? path : path + '/'
-
       // For now assumes /orders/ path. Needs logic to try all types for a valid response:
       // Orders, transactions, tokens, batches
-      query && query.length > 0 && history.push(`${pathPrefix}orders/${query}`)
+      query && query.length > 0 && history.push(`/orders/${query}`)
```

Comment:
AFAIR the idea for this was to keep the network on the path: /xdai/orders/dasdasd

I tested and when you search (and don't find) something the path will change to `/orders/adsdkjha`

Not the biggest deal since when the order is found it'll revert to the right one, so I'd be ok removing it.

Thoughts @anxolin ?